### ISSUE TITLE
sometimes plays audio, not video when continues tap on next button

### ISSUE THREAD

**Issue by anilappvolution** [0xced/XCDYouTubeKit#245]
When tap on next button in player. I assigned new video id to player so when continuos tap on next 3 or more time then only play audio not display video. please help me.

Thanks in Advance.

**Comment by dhavalappvolution** [comment]
yes I'm getting same issue. Please let me know if you'll find any solution for this.

**Comment by randikachan** [comment]
Seems like this issue is not resolved yet! :(

What I noticed is as follows:

I have embedded the `XCDYouTubeVideoPlayerViewController` inside a view in my ViewController.
When I pop that ViewController, in its `- (void)viewWillDisappear:(BOOL)animated` I stop the moviePlayer as follows:

       `[self.videoPlayerViewController.moviePlayer stop];`

But when the app has gone to the previous VC from the Popped VC, it starts to play the Audio track of the previously loaded Video. :(

**Hint:** What I noticed is, it doesn't work the `[moviePlayer stop]` method call if we do it just before the video is actually loaded.

I'm also looking for an solution for this issue.
